Sources/SwiftCurrent_SwiftUI/Views/WorkflowView.swift

```diff
@@ -0,0 +1,214 @@
+///             $0.background(Color.gray)
+///                 .transition(.slide)
+///                 .animation(.spring())
+///         }
+///     })
+///     .thenProceed(with: WorkflowItem(SecondView.self)
+///                     .persistence(.removedAfterProceeding)
+///                     .applyModifiers {
+///         if true {
+///             $0.SecondViewSpecificModifier()
+///                 .padding(10)
+///                 .background(Color.purple)
+///                 .transition(.opacity)
+///                 .animation(.easeInOut)
+///         }
+///     })
+///     .onAbandon { print("presentingWorkflowView is now false") }
+///     .onFinish { args in print("Finished 1: \(args)") }
+///     .onFinish { print("Finished 2: \($0)") }
+///     .background(Color.green)
+///  ```
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+public struct WorkflowView<Args>: View {
+    @Binding private var isPresented: Bool
+    @StateObject private var model = WorkflowViewModel()
+    @State private var didLoad = false
+
+    let inspection = Inspection<Self>() // Needed for ViewInspector
+    private var workflow: AnyWorkflow?
+    private var onFinish = [(AnyWorkflow.PassedArgs) -> Void]()
+    private var onAbandon = [() -> Void]()
+    private var passedArgs = AnyWorkflow.PassedArgs.none
+
+    /// Creates a `WorkflowView` that displays a `FlowRepresentable` when presented.
+    public init(isPresented: Binding<Bool>) where Args == Never {
+        _isPresented = isPresented
+    }
+
+    /// Creates a `WorkflowView` that displays a `FlowRepresentable` when presented.
+    public init(isPresented: Binding<Bool>, args: Args) {
+        _isPresented = isPresented
+        if let args = args as? AnyWorkflow.PassedArgs {
+            passedArgs = args
+        } else {
+            passedArgs = .args(args)
+        }
+    }
+
+    private init(isPresented: Binding<Bool>,
+                 workflow: AnyWorkflow?,
+                 onFinish: [(AnyWorkflow.PassedArgs) -> Void],
+                 onAbandon: [() -> Void],
+                 passedArgs: AnyWorkflow.PassedArgs) {
+        _isPresented = isPresented
+        self.workflow = workflow
+        self.onFinish = onFinish
+        self.onAbandon = onAbandon
+        self.passedArgs = passedArgs
+    }
+
+    public var body: some View {
+        if isPresented {
+            VStack {
+                model.body
+            }
+            .onAppear {
+                guard !didLoad else { return }
+                didLoad = true
+                model.isPresented = $isPresented
+                model.onAbandon = onAbandon
+                workflow?.launch(withOrchestrationResponder: model,
+                                 passedArgs: passedArgs,
+                                 launchStyle: .new) { passedArgs in
+                    onFinish.forEach { $0(passedArgs) }
+                }
+            }
+            .onDisappear {
```

Comment:
I think we need to reset the didLoad here back to false.  The reason is that if you toggle showing the workflow, we end up failing to show it when we toggle back on.
```suggestion
            .onDisappear {
                didLoad = false
```